Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -1617,8 +1609,8 @@ namespace Remotion.BocAutoCompleteReferenceValue
             LayoutUtility.Hide(this.element);
             this.input.closest('div, td, th, body')!.appendChild(this.element);
 
-            if (this.options.combobox.getAttribute('aria-labelledby') !== null) {
-                this.element.setAttribute("aria-labelledby", this.options.combobox.getAttribute('aria-labelledby')!);
+            if (this.input.getAttribute('aria-labelledby') !== null) {
+                this.element.setAttribute("aria-labelledby", this.input.getAttribute('aria-labelledby')!);
```

Comment:
wait a second, I think this bit of code can simply go away? orginally, it was because I already rendered the aria-labelledby onto the input box and then moved it up to the span. what is this code doing now? please check and either explain or remove because redundant.

Reply:
This code relates to the info-box and moves the `aria-labelledby` to the opened infobox. I do think that this is still relevant.

Reply:
Then we're done.